starter-code/functions-and-arrays.js

```diff
@@ -124,10 +159,10 @@ var words = [
 ];
 
 var howManyMatter = howManyTimes(words, "matter");
-console.log(howManyMatter);
+console.log("How many words (matter):" + howManyMatter);
 
 var howManyDog = howManyTimes(words, "dog");
-console.log(howManyDog);
+console.log("How many words (dog):" + howManyDog);
 
 // Bonus Quest
 function greatestProduct (matrix) {
```

Comment:
¿Que ha pasado? ¿No habéis podido salir de Matrix? Es broma chicas!!! Excelente trabajo!!